### PR TITLE
fix: admins should see all spaces when moving items

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: https://github.com/lightdash/lightdash/issues/6876

### Description:

CleanShot 2023-09-04 at 13 30 42@2x
CleanShot 2023-09-04 at 13 30 54@2x
CleanShot 2023-09-04 at 13 30 56@2x

### CODE REVIEW

**Comment by IrakliJani** [comment]
@TuringLovesDeathMetal should we distinguish "admin content spaces" somehow in the menu?
im worried about the clutter admins might get 🤔

File: packages/frontend/src/components/common/ResourceView/ResourceActionMenu.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const ResourceViewActionMenu: FC<ResourceViewActionMenuProps> = ({
     const location = useLocation();
     const { projectUuid } = useParams<{ projectUuid: string }>();
     const organizationUuid = user.data?.organizationUuid;
-    const { data: spaces = [] } = useSpaceSummaries(projectUuid);
+    const { data: spaces = [] } = useSpaceSummaries(projectUuid, true);
```
Comment (joaoviana) [inline packages/frontend/src/components/common/ResourceView/ResourceActionMenu.tsx:54]: shouldn't `true` be instead `user.data.ability.can(something-that-indicates-the-user-is-admin)`?

**Review by joaoviana — CHANGES_REQUESTED** [review]

File: packages/frontend/src/components/common/ResourceView/ResourceActionMenu.tsx (hunk 1 above)
Comment (IrakliJani) [reply packages/frontend/src/components/common/ResourceView/ResourceActionMenu.tsx:54]: @joaoviana that part is handled on the backend

**Review by joaoviana — APPROVED** [review]

**Comment by PriPatel** [comment]
@IrakliJani 

> should we distinguish "admin content spaces" somehow in the menu?
> im worried about the clutter admins might get 🤔

Just came here to comment on this! I think using subheadings that match the tabs in the Spaces page would help! (grouped headers, one saying 'Shared with me' and another saying 'admin content view')  We can update copy later but for now it should match the tabs in the Spaces page.

**Pull request merged**